src/RadixDlt.NetworkGateway.PrometheusIntegration/GatewayApiMetricObserver.cs

```diff
@@ -169,6 +174,11 @@ internal class GatewayApiMetricObserver :
             "Number of mempool transactions submitted to the gateway which were already being tracked"
         );
 
+    public void OnSqlQueryExecuted(string queryName, TimeSpan duration)
+    {
+        _sqlQueryDuration.WithLabels(queryName).Observe(duration.TotalMilliseconds);
```

Comment:
I see we're using default histogram buckets, therefore we should use duration.TotalSeconds over here.